task-sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -1238,6 +1238,8 @@ def ensure_secrets_backend_loaded() -> list[BaseSecretsBackend]:
 
     backends = ensure_secrets_loaded(default_backends=DEFAULT_SECRETS_SEARCH_PATH_WORKERS)
 
+    log = structlog.get_logger(logger_name="supervisor")
```

Comment:
structlog is lazy, so I don't think this is any different from the `log` we have on L117.

Does this actually fix it?

Reply:
Yeah seems to work

Reply:
Oh also: we say this is the supervisor logger, but the call stack shows it coming from the DAG code. THat's not great.

But all of that is a nit, lets not get hung up about it

Reply:
Alright yeah but i am getting the logger from `structlog.get_logger(logger_name="supervisor")`. Is it wired in that way?